### ISSUE TITLE
[Adaptive Cards iOS - Flight Itinerary]: Voice over does not convey the complete information about the days under the flight itinerary screen.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#7576]
### Target Platforms

iOS

### SDK Version

Version 1.0 (2.3.1-beta.20220620.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 15.5
Application: Adaptive Cards
App Version: 1.0 (2.4.0-beta.20220612.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.0" control and select it.
5. Navigate to 'Scenarios' control and activate it.
6. Navigate to 'FlightItinerary.json' and activate it.
7. Navigate to the Day and observe the issue.

**Actual Result:**
Voice over does not convey the complete days information to the user while navigating to the days as it is announcing as **'Tue comma May 30 comma two thousand seventeen 12 colon 25 PM'**

**Expected Result:**
While navigating to travel day information, screen reader should convey the complete information about the day and it should announce as **'Tuesday comma May 30 comma two thousand seventeen 12 colon 25 PM'**

**User Experience:**
It will impact the screen reader user if the voice over does not convey the complete information for the day.

**MAS Reference:**
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/01.03.01

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/95913748/176207866-802a7123-9935-43e4-9985-ce12bf4e5491.MP4

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
Closing this issue here as of now this issue is being tracked with below URL.

https://github.com/microsoft/AdaptiveCards-Mobile/issues/40
